src/components/ForecastScreen.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import ForecastList from './ForecastList';
+
+function ForecastScreen(props) {
```

Comment:
Same thing as CityListScreen. This one is not necessary, all the stuff can go directly in `ForecastList` (which I again advise to be renamed into `ForecastScreen`).